default/stringtables/en.txt

```diff
@@ -4705,6 +4705,29 @@ Important sections include [[encyclopedia CATEGORY_GAME_CONCEPTS]] and the recen
 
 '''
 
+SITREP_DISMISS_HIDE_TITLE
+Dismissing and Hiding SitReps
+
+SITREP_DISMISS_HIDE_SHORT_DESC
+Dismissing a specific SitRep and hiding a SitRep template.
```

Comment:
Thanks, I kept trying to use 'group' as a replacement.  Type seems a better fit to a user, though slightly less accurate.